Cargo.lock

```diff
@@ -2074,6 +2083,21 @@ version = "0.1.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "448296241d034b96c11173591deaa1302f2c17b56092106c1f92c1bc0183a8c9"
 
+[[package]]
+name = "ring"
```

Comment:
CI failed because of this.

Reply:
Should I switch to another HTTP client library?

Reply:
I think it's preferrable. I checked `cargo-deny`, but there was no option to exclude build-time dependencies.